src/config/constants.js

```diff
@@ -117,6 +118,145 @@ export const REDIRECT_URL_LOCAL_STORAGE_KEY = 'redirectUrl';
 
 export const EMPTY_ROW_HEIGHT = 53;
 
+export const TABLE_TYPES = {
+  MEMBER: 'member',
+  ITEM: 'item',
+  PERMISSION: 'permission',
+  ADMIN: 'admin',
+};
+
+export const memberHeadCell = [
+  {
+    id: 'name',
+    numeric: false,
+    label: 'Name',
+    align: 'left',
+  },
+  {
+    id: 'email',
+    numeric: false,
+    label: 'Email',
+    align: 'right',
+  },
+  {
+    id: 'type',
+    numeric: false,
+    label: 'Type',
+    align: 'right',
+  },
+  {
+    id: 'createdAt',
+    numeric: false,
+    label: 'Created At',
+    align: 'right',
+    type: ITEM_DATA_TYPES.DATE,
+  },
+  {
+    id: 'updatedAt',
+    numeric: false,
+    label: 'Updated At',
+    align: 'right',
+    type: ITEM_DATA_TYPES.DATE,
+  },
+];
+
+export const adminsHeadCell = [
+  {
+    id: 'name',
+    numeric: false,
+    label: 'Name',
+    align: 'left',
+  },
+  {
+    id: 'roles',
+    numeric: false,
+    label: 'Roles',
+    align: 'center',
+  },
+  {
+    id: 'email',
+    numeric: false,
+    label: 'Email',
+    align: 'right',
+  },
+  {
+    id: 'type',
+    numeric: false,
+    label: 'Type',
+    align: 'right',
+  },
+  {
+    id: 'createdAt',
+    numeric: false,
+    label: 'Created At',
+    align: 'right',
+    type: ITEM_DATA_TYPES.DATE,
+  },
+  {
+    id: 'updatedAt',
+    numeric: false,
+    label: 'Updated At',
+    align: 'right',
+    type: ITEM_DATA_TYPES.DATE,
+  },
+];
+
+export const itemHeadCells = [
+  {
+    id: 'name',
+    numeric: false,
+    label: 'Name',
+    align: 'left',
+  },
+  {
+    id: 'type',
+    numeric: false,
+    label: 'Type',
+    align: 'right',
+  },
+  {
+    id: 'ownerName',
+    numeric: false,
+    label: 'Owner',
+    align: 'right',
+  },
+  {
+    id: 'createdAt',
+    numeric: false,
+    label: 'Created At',
+    align: 'right',
+    type: ITEM_DATA_TYPES.DATE,
+  },
+  {
+    id: 'updatedAt',
+    numeric: false,
+    label: 'Updated At',
+    align: 'right',
+    type: ITEM_DATA_TYPES.DATE,
+  },
+];
+
+export const permissionHeadCells = [
+  {
+    id: 'description',
+    numeric: false,
+    label: 'Description',
+    align: 'left',
+  },
+  {
+    id: 'endpoint',
+    numeric: false,
+    label: 'Endpoint',
+    align: 'left',
+  },
+  {
+    id: 'method',
+    numeric: false,
+    label: 'Method',
+    align: 'left',
+  },
+];
+
```

Comment:
Do you think it would be better to have a `tableContants` file to separate it from all other constants?